lib/core/postcss.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require("path");
+module.exports = {
+  plugins: {
+    "postcss-import": {
+      resolve(id, basedir) {
+        // resolve alias @css, @import '@css/style.css'
+        // because @css/ has 5 chars
+        if (id.startsWith("@css")) {
+          return path.resolve("./src/assets/styles/css", id.slice(5));
+        }
+
+        // resolve node_modules, @import '~normalize.css/normalize.css'
+        // similar to how css-loader's handling of node_modules
+        if (id.startsWith("~")) {
+          return path.resolve("./node_modules", id.slice(1));
+        }
+
+        // resolve relative path, @import './components/style.css'
+        return path.resolve(basedir, id);
+      }
+    },
+    "postcss-simple-vars": {},
+    "postcss-nested": {},
+    "postcss-url": {},
+    autoprefixer: {
+      overrideBrowserslist: "> 1%, IE 6, Explorer >= 10, Safari >= 7"
+    }
+  }
+};
```

Comment:
We could use [PostCSS Preset Env ](https://github.com/csstools/postcss-preset-env) that already includes most of features like CSS Variables, Nesting, Autoprefixer, etc. out of the box.

Reply:
Already used it in rollup build, see [https://github.com/moaijs/moai-vue/blob/8427e2286f592244ff66364f3b65ce62c690b7e4/lib/core/rollup.config.js#L9](https://github.com/moaijs/moai-vue/blob/8427e2286f592244ff66364f3b65ce62c690b7e4/lib/core/rollup.config.js#L9)

Reply:
so are we good here @ng-hai ?

Reply:
No, I mean we won't need to install these plugins if applied `postcss-preset-env`
- postcss-simple-vars
- postcss-nested 
- autoprefixer

Reply:
Yeah I think we will deal with CSS/Styling in another PR :)) just a JS-only build is good enough for the first PR